src/ui/server/helpers/page-variables/single-input/index.test.ts

```diff
@@ -14,6 +14,7 @@ describe('server/helpers/page-variables/single-input', () => {
     },
     BACK_LINK: '/mock',
     FEEDBACK_ROUTE: feedbackRoute,
+    CONDITIONAL_YES_HTML: 'testpage/page.njk',
```

Comment:
constants or named as something more inline with a conditional component (i.e not a page)?